scripts/dockerfiles/monitrc.erb

```diff
@@ -14,3 +14,6 @@ include /etc/monit/monitrc.d/rsyslog
 
 include /var/vcap/monit/*.monitrc
 include /var/vcap/monit/job/*.monitrc
+
+# Standard pseudo-job. Triggered to mark the role as ready.
+check file ready path /tmp/ready
```

Comment:
Sort of kind of both? We just [install it from Ubuntu](https://github.com/hpcloud/fissile/blob/master/scripts/dockerfiles/Dockerfile-base#L15).  The only reason I'm asking that it's not in `/tmp` is to make it easier to find (since everything else lives in `/var/vcap`).